assets/js/components/Collapsible.js

```diff
@@ -0,0 +1,182 @@
+'use strict';
+
+const utils = require('../libs/elife-utils')();
+
+module.exports = class Collapsible {
+
+  constructor($elm, _window = window, doc = document) {
+
+    this.$elm = $elm;
+    this.window = _window;
+    this.doc = doc;
+
+    this.viewportWidthMedium = 729;
+    this.viewportWidthLarge = 899;
```

Comment:
It is my understanding that we need the 2 view port sizes to amend the text on narrower screens still. Can we push back on this to simplify the js?

@chris-huggins to avoid the additional logic can we just agree on the same text for narrow and medium?